connectors/redis/redigo_test.go

```diff
@@ -73,8 +75,39 @@ func TestWrongPort(t *testing.T) {
 		Host: "localhost",
 		Port: 1111,
 	}
-	c := redis.NewRedigoClient(config)
+	c := redis.NewRedigoClient(config, nil)
 	_, err := c.Get("testkey")
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), "1111")
 }
+
+// Test that we track latencies for all the redis commands
+func TestTimerCalled(t *testing.T) {
+	if !redis.IsRunning() {
+		t.SkipNow()
+	}
+
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	stats := mocks.NewMockScope(ctrl)
+
+	ctrl2 := gomock.NewController(t)
+	defer ctrl2.Finish()
+	timer := mocks.NewMockTimer(ctrl2)
+
+	c := redis.NewRedigoClient(testRedisConfig.ServerSettings, stats)
+
+	redisCommands := map[string]func(){
+		"GET": func() { c.Get("a") },
+		"SET": func() { c.SetEx("a", []byte{1}, 9*time.Second) },
+		"DEL": func() { c.Del("a") },
+	}
+	for command, f := range redisCommands {
+		stats.EXPECT().SubScope("redis").Return(stats)
```

Comment:
nit: run each of these as a seperate test using `t.Run`. E.g. https://github.com/uber-go/dosa/blob/master/connectors/cassandra/schema_test.go#L142